jetty-ee10/jetty-ee10-annotations/src/test/resources/web-fragment4true.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<web-fragment xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-fragment_4_0.xsd" metadata-complete="true" version="4.0">
+<web-fragment
+  xmlns="https://jakarta.ee/xml/ns/jakartaee"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/web-fragment_6_0.xsd"
+  metadata-complete="true"
+  version="6.0">
```

Comment:
fragment4 suggests this should be a 4.0 file

Reply:
Dang, good point on all 3 of these.

This was a mechanical change, but I didn't consider the filenames themselves.

Reply:
This original XML is from Servlet 4.0, that's not supported anymore while on Servlet 6.0.
Should this be upgraded to be "old" on Servlet 5.0 at least?